vispy/gloo/context.py

```diff
@@ -166,7 +166,12 @@ def flush_commands(self, event=None):
         """
         if self._do_CURRENT_command:
             self._do_CURRENT_command = False
-            self.shared.parser.parse([('CURRENT', 0)])
+            canvas = get_current_canvas()
+            if canvas and hasattr(canvas, '_backend'):
+                fbo = canvas._backend._vispy_get_fb_bind_location()
+            else:
+                fbo = 0
+            self.shared.parser.parse([('CURRENT', 0, fbo)])
```

Comment:
Ah, I didn't know about that wiki page. It says:
> `<ID>` is an integer unique within the current GL context that is used as a reference to a GL object.

If I read that correctly, I think that means I can just replace the hard-coded `0` with the fbo?